packages/@aws-cdk/cloud-assembly-schema/lib/manifest.ts

```diff
@@ -0,0 +1,336 @@
+/**
+ * Common properties for asset metadata.
+ */
+interface BaseAssetMetadataEntry {
```

Comment:
Meh :\

I don't have a good clear phrase for this, partly because we actually abused this property. Lets just keep this for now as is, it will soon be replaced by a separate schema for assets.